flink-streaming-connectors/flink-connector-kafka/src/test/java/org/apache/flink/streaming/connectors/kafka/KafkaITCase.java

```diff
@@ -40,6 +40,11 @@ public void testCheckpointing() throws Exception {
 		runCheckpointingTest();
 	}
 
+	@Test(expected = RuntimeException.class)
```

Comment:
You don't check the exception here. This means any `RuntimeException` can make the test pass. IMO not a good test.